getting-started/deployment.md

```diff
@@ -6,6 +6,10 @@ Deploying a Solidus application to production is no different from deploying any
 
 ### Heroku
 
+{% hint style="warning" %}
+Heroku [does not support](https://devcenter.heroku.com/articles/sqlite3) the rails default database sqlite3. Heroku natively utilizes [PostgreSQL](https://www.postgresql.org/) and adaptively uses other [data storage services](https://elements.heroku.com/addons).
```

Comment:
Let's use `Rails` with capital `R`. Also,`Sqlite3` with capital `S`.

I think we should provide a solution along with the problem. For example, we may want to suggest changing the database for production only:

```
# In Gemfile:

group :development, :test do
  gem 'sqlite3'
end

group :production do
  gem 'pg'
end

# In config/database.yml

production:
  adapter: postgresql
  encoding: unicode
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
  database: myapp_production
  username: myapp
  password: <%= ENV['MYAPP_DATABASE_PASSWORD'] %>

```

Reply:
Updated proper noun capitalization and fixed "Awesome" to "Amazing." Thank you for catching that!